oak-commons/src/main/java/org/apache/jackrabbit/oak/commons/io/LazyInputStream.java

```diff
@@ -48,7 +47,7 @@ public LazyInputStream(Supplier<InputStream> inputStreamSupplier) {
      * @deprecated Use {@link #LazyInputStream(Supplier)} instead
      */
     @Deprecated(since = "1.20.0", forRemoval = true)
-    public LazyInputStream(ByteSource byteSource) {
+    public LazyInputStream(com.google.common.io.ByteSource byteSource) {
```

Comment:
Shouldn't this be from new jar? If not, then we should revert the changes in this file.